.docker/setup_config.sh

```diff
@@ -1,6 +1,15 @@
 set -e
 echo "About to start conda update, this may take some time..."
-source setup/setup_conda.sh Linux-x86_64
+ARCH=$(uname -m)
+if [ "$ARCH" == "x86_64" ]; then
+    source setup/setup_conda.sh Linux-x86_64
+elif [ "$ARCH" == "aarch64" ]; then
+    source setup/setup_conda.sh Linux-aarch64
+else
+    echo "Unsupported architecture: $ARCH"
+    exit 1
+fi
+
```

Comment:
I am very skeptical that this is actually broken:
- GitHub actions uses an arm chip for macos-latest https://github.com/actions/runner-images/tree/main
<img width="922" alt="Screenshot 2024-10-18 at 5 32 18 PM" src="https://github.com/user-attachments/assets/8d8690be-839f-47d6-8db6-4674271e2b6f">
- We use macos-latest for our CI/CD build 
https://github.com/e-mission/e-mission-server/blob/master/.github/workflows/osx-ubuntu-manual-install.yml
<img width="409" alt="Screenshot 2024-10-18 at 5 34 00 PM" src="https://github.com/user-attachments/assets/d87749b5-76f2-48fd-804f-bb77901ddd91">
- The CI/CD build is passing
https://github.com/e-mission/e-mission-server/actions/workflows/osx-ubuntu-manual-install.yml

Before I review this further, I would like some additional detailed evidence that this is actually broken and not just an error in your installation.